stac_validator/stac_utilities.py

```diff
@@ -107,7 +107,7 @@ def item_schema_url(cls, version, filename="stac-item.json"):
         return cls(version, "json-schema", filename).ITEM_URL
 
     @classmethod
-    def item_geojson_schema_url(cls, version, filename="geojson.json"):
+    def geojson_schema_url(cls, version, filename="geojson.json"):
```

Comment:
`geojson_schema_url` name looks more to be a property than a method

Reply:
prepend with `get_`?